### PR TITLE
Plamen/active trip

### PR DESCRIPTION
**Pull request by PlamenMichev**
1. Add the plan trip page
2. Use the pinboards and pins logic
3. Add drag and drop functionality
4. Style the dropping
5. Connect plan trip with the other pages

STILL NOT DONE:
1. The drag and drop works only for the first `Drag pin here` for the morning
2. There is no way to add pins to pinboards currently(should add them manually)
3. Styling of the dropped pins
4. The page opens directly with the pinboards, there is no swiping functionality to discover them

Review the changes so far and we will see what I can do more until the demo

image

### CODE REVIEW

File: src/components/AvailablePinBoards/index.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      {/* <Draggable index={3} draggableId={'forth-id'}>
+        {(provided) => (
+          <div ref={provided.innerRef} {...provided.draggableProps} {...provided.dragHandleProps}>
+            <PinBoard
+              title={'Title 4'}
+              images={availablePinboards.slice(9, 12).map((x) => x.images[0])}
+            />
+          </div>
+        )}
+      </Draggable> */}
```
Comment (vexleet) [inline src/components/AvailablePinBoards/index.jsx:63]: Delete if not needed

File: src/components/PinBoard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import './index.styles.css';
+import PropTypes from 'prop-types';
+
+const PinBoard = ({ pins, title }) => {
```
Comment (vexleet) [inline src/components/PinBoard/index.jsx:4]: I have also created a PinBoard (called `Pingboard`) component so we have to see which one to use.

**Comment by vexleet** [comment]
Im doing that:
`There is no way to add pins to pinboards currently(should add them manually)`

What do you mean by that `The drag and drop works only for the first Drag pin here for the morning`? That after you first drag, you cant drag anymore or that you can only drag and drop in the morning tab?

**Comment by PlamenMichev** [comment]
> Im doing that: `There is no way to add pins to pinboards currently(should add them manually)`
> 
> What do you mean by that `The drag and drop works only for the first Drag pin here for the morning`? That after you first drag, you cant drag anymore or that you can only drag and drop in the morning tab?

Basically you can test the functionality only for the first slot now. I have to do the second slot to work and also the afternoon and evening slots. I didn't do them now because I wanted to agree with you if the flow is fine, if it is, I will just apply the same logic for the other slots

**Comment by vexleet** [comment]
> > Im doing that: `There is no way to add pins to pinboards currently(should add them manually)`
> > What do you mean by that `The drag and drop works only for the first Drag pin here for the morning`? That after you first drag, you cant drag anymore or that you can only drag and drop in the morning tab?
> 
> Basically you can test the functionality only for the first slot now. I have to do the second slot to work and also the afternoon and evening slots. I didn't do them now because I wanted to agree with you if the flow is fine, if it is, I will just apply the same logic for the other slots

LGTM so you can do it when you have the time.

**Pull request merged**